### ISSUE TITLE
Viewport position in absolute screen coordinates

### ISSUE THREAD

**Issue by Fredosixx** [SketchUp/api-issue-tracker#819]
It would be useful to have a **method to get the [top, left] coordinate of the viewport**.

This would allow to accurately position web dialogs within the viewport, for instance, by reference to the position of objects or the mouse.

**Comment by DanRathbun** [comment]
Yes, with a `view#vporigin` and the existing [`#vpheight`](https://ruby.sketchup.com/Sketchup/View.html#vpheight-instance_method) and [`#vpwidth`](https://ruby.sketchup.com/Sketchup/View.html#vpwidth-instance_method) we could better position things, especially when drawing to the view, positioning model notes, etc.
